libs/ui-watt/src/lib/components/input/datepicker/watt-datepicker.component.ts

```diff
@@ -61,6 +62,7 @@ const danishLocaleCode = 'da';
     WattInputMaskService,
     WattRangeInputService,
     { provide: MatFormFieldControl, useExisting: WattDatepickerComponent },
+    MAT_DATEPICKER_SCROLL_STRATEGY_FACTORY_PROVIDER,
```

Comment:
question: seems like this provider is already provided in the `MatDatepickerModule` (https://github.com/angular/components/blob/main/src/material/datepicker/datepicker-module.ts#L85). I wonder why is it needed again. Any idea?